client/components/data/query-billing-data/index.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingBillingData } from 'state/billing-data/selectors';
+import { requestBillingData } from 'state/billing-data/actions';
+
+class QueryBillingData extends Component {
+	static propTypes = {
+		requestingBillingData: PropTypes.bool,
+		requestBillingData: PropTypes.func
+	};
+
+	componentWillMount() {
+		this.request( this.props );
+	}
+
+	request( props ) {
```

Comment:
If this is only used once, I think it's fine to have the contents of this function inside of componentWillMount eg:

```javascript
componentWillMount() {
    if ( this.props.requestingBillingData ) {
        return;
    }
    this.props.requestBillingData();
}
```